spec/features/users/sign_up_spec.rb

```diff
@@ -376,14 +376,8 @@ def clipboard_text
     visit_idp_from_oidc_sp_with_hspd12_and_require_piv_cac
     sign_up_and_set_password
 
-    expect(page).to_not have_selector('#two_factor_options_form_selection_phone', count: 1)
-    expect(page).to_not have_selector('#two_factor_options_form_selection_webauthn', count: 1)
-    expect(page).to_not have_selector('#two_factor_options_form_selection_auth_app', count: 1)
-    expect(page).to_not have_selector('#two_factor_options_form_selection_backup_code', count: 1)
-    expect(page).to have_selector('#two_factor_options_form_selection_piv_cac', count: 1)
-
-    select_2fa_option('piv_cac')
-    expect(page).to_not have_content(t('two_factor_authentication.piv_cac_fallback.question'))
```

Comment:
At one point it seems that we may have prevented the user from returning to the MFA selection during account setup during HSPD12, but as above, the `to_not` wasn't doing anything meaningful here, and we've shown an option to return to the MFA selection for at least a while now. But the MFA selection does prevent the user from choosing anything other than PIV.

![Screen Shot 2023-07-24 at 8 34 35 AM](https://github.com/18F/identity-idp/assets/1779930/8438c015-7f7f-47c9-85e6-56ea4c6bb59f)
![Screen Shot 2023-07-24 at 8 34 41 AM](https://github.com/18F/identity-idp/assets/1779930/4ae989f8-d1a9-4073-b965-060d3134d5b4)